Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -133,7 +133,7 @@ public function testGetGroupedQueries(): void
     /** @group legacy */
     public function testGetGroupedQueriesWithDeprecatedDebugStackLogger(): void
     {
-        if (Kernel::VERSION_ID >= 70000) {
```

Comment:
symfony/framework-bundle 6.4 and symfony/web-profiler-bundle 7.0 were installed so this condition did not work